middleware/securityHeaders.js

```diff
@@ -1,56 +1,67 @@
 'use strict';
 const helmet = require('helmet');
+const { map } = require('lodash');
+const { legacyProxiedRoutes } = require('../controllers/routes');
 
-// these URLs won't get the helmet header protection
-// @TODO this should only affect the legacy homepage
-const pathsExemptFromHelmet = ['/', '/welsh', '/legacy', '/funding/funding-finder', '/welsh/funding/funding-finder'];
+module.exports = function(app) {
+    /**
+     * URLs which should be exempt from security headers
+     * Only proxied legacy URLs should be exempt.
+     */
+    const exemptLegacyUrls = map(legacyProxiedRoutes, _ => _.path);
```

Comment:
Should this "welshify" these URLs too?

Reply:
Noting down what we discussed that, no it doesn't need to welshify as we're using legacy urls as a per-URL opt-in so preferable to not double up urls automatically. Also, worked out that the `/funding/funding-finder` proxy code can go as it's been superseded by `/funding/programmes`